.travis.yml

```diff
@@ -0,0 +1,13 @@
+os: osx
+osx_image: xcode11
+
+before_install:
+  - brew update
+  - brew upgrade python3 || true
+  - brew install scons git doxygen || true
+  - pip3 install modm
+
+script:
+  - cd examples/darwin/logger
+  - lbuild build
```

Comment:
We could also change the `compile_examples.py` to use `lbuild -D:target=hosted-darwin build` for the linux examples and only compile those. All linux examples should also compile for macOS (I test them that way).